k8s/overlays/nerc-ocp-infra/patches/bc-xdmod.yaml

```diff
@@ -0,0 +1,9 @@
+apiVersion: build.openshift.io/v1
+kind: BuildConfig
+metadata:
+  name: bc-xdmod
+spec:
+  source:
+    type: Git
+    git:
+      ref: ArgoDeploy
```

Comment:
Should this be updated to `main` before we merge?

Reply:
This should be updated to the specific commit that we will use to deploy from